cpu/stm32/stmclk/stmclk_l4wx.c

```diff
@@ -643,5 +643,14 @@ void stmclk_init_sysclk(void)
         gpio_init_af(GPIO_PIN(PORT_A, 8), GPIO_AF0);
     }
 
+#if IS_USED(MODULE_PERIPH_USBDEV)
+    /* USBDEV is clocked by HSI48 so enable this clock when the peripheral is used */
```

Comment:
According to the clock tree (RM0434, page 205), the USB device can be clocked by MSI, HSI48, PLLQ or PLLSAI1Q. And all must be 48MHz. Also note that on some L4 (L47x, L48x), there's no HSI48 internal RC
So we should extend the clock configuration to allow selecting the source and ensure the source is 48MHz. But let's leave this to a follow-up.